test/browser/home-page.test.jsx

```diff
@@ -0,0 +1,30 @@
+var should = require('should');
+var React = require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var stubContext = require('./stub-context.jsx');
+var stubBlogFeedLoader = require('./stub-blog-feed-loader.js');
+var HomePage = require('../../pages/home.jsx');
+
+describe("HomePage", function() {
+  var homePage, blogSection;
+
+  beforeEach(function() {
+    homePage = stubContext.render(HomePage);
+    blogSection = stubContext.render(HomePage.BlogSection);
+  });
+
+  describe("blogSection", function() {
+    it("should display featured post", function() {
+      blogSection.getDOMNode().querySelector(".featured-post .entry-title").textContent.should.not.eql("");
+    });
+    it("should display 3 other latest posts", function() {
+      blogSection.getDOMNode().querySelectorAll(".recent-posts .post-title").length.should.equal(3);
+    });
```

Comment:
Oh, you don't want to do anything asynchronous within `describe()`--only within `it()`, and when you do that, you want to pass it a callback that takes a [`done` argument](http://mochajs.org/#asynchronous-code). If you don't do that, mocha will think your test is synchronous, and start tearing down (unmounting) your test setup before the real part of the test has even started!